data_steward/tools/create_rdr_snapshot.py

```diff
@@ -5,12 +5,10 @@
 from datetime import datetime
```

Comment:
check on your side, but I think this import can be removed entirely.

Reply:
I found an removed an additional unused import

Reply:
was the change pushed?